command/commands/user.py

```diff
@@ -43,7 +43,8 @@ def init_subparsers(self) -> _SubParsersAction:
         parser_view.set_defaults(which="view",
                                  help="View information about a given user.")
         parser_view. \
-            add_argument("--slack_id", type=str, action='store',
+            add_argument("--slack-id", metavar="SLACK-ID",
```

Comment:
I am reading the docs and it looks like there is simply no mention of `--slack-id`, even though the code references it. @rwblickhan do you think we should remove the two dashes before `--slack-id` so that it is treated as a variable instead of a flag?

Docs in question: https://rocket2.readthedocs.io/en/latest/docs/UserCommands.html#view

Reply:
Hmm, yes, seems the implementation diverged a bit from the spec. That being said, I actually like the implementation better - I can do `/rocket user view` just to view my own profile, or `/rocket user view --slack-id @user` to specify another one. If there's not a good argument against, I would say the docs should be changed, instead.